packages/wallet/test/integration/cip30mapping.test.ts

```diff
@@ -475,8 +476,11 @@ describe('cip30', () => {
           Cardano.Address.fromBytes(HexBlob(cipAddr)).toBech32()
         );
 
-        const [{ rewardAccount: walletRewardAccount }] = await firstValueFrom(wallet.addresses$);
-        expect(cipRewardAddresses).toEqual([walletRewardAccount]);
+        const walletAddresses = await firstValueFrom(wallet.addresses$);
+        const rewardAccounts = uniq(walletAddresses.map((address) => address.rewardAccount));
```

Comment:
To make sure that the change is covered by tests
```suggestion
        const rewardAccounts = uniq(walletAddresses.map((address) => address.rewardAccount));
        expect(rewardAccounts.length).toBeGreaterThan(1);
```

Reply:
fixed in [f3e4588](https://github.com/input-output-hk/cardano-js-sdk/pull/1044/commits/f3e4588394670c63c91d8ebdc9665c7cabbd7b58)

Reply:
@mchappell the new assertion is failing, probably need to update test setup

Reply:
Updated in [98c9e41](https://github.com/input-output-hk/cardano-js-sdk/pull/1044/commits/98c9e41b8c82fd4dbadcf3606cf32c2ea5bfb9d3) to use greaterThanOrEqual as we don't need to guarantee it'll be higher than 1 result, unless we want to change the test so that `walletAddresses` is a property greater than 1?

Reply:
This is fine for this test, but the idea of behind this change request was to have test coverage of the change, which isn't achieved when length is 1